_includes/schema-index.html

```diff
@@ -7,7 +7,7 @@
     "url": "{{ site.url }}",
     "description": "{{ site.description }}",
     "disambiguatingDescription": "{{ site.abstract }}",
-    "keywords": ["time tracker", "time tracking", "software", "app", "screen time", "free", "open source", "automatic", "self-hosted"],
+    "keywords": ["time tracker", "time tracking", "screen time tracker", "open source screen time tracker", "opensourcescreentimetracker", "free open source time tracker", "freeopensourcetimetracker", "laptop screen time tracker", "automated time tracker", "best free open source software", "bestfreeopensourcesoftware"],
```

Comment:
Could you explain the reasoning here? What makes the ones that were added/removed good/bad?